### DISCUSSION TITLE
Some ideas from the JuMP-dev workshop

### DISCUSSION THREAD

**Discussion by datejada** [TulipaEnergy/TulipaEnergyModel.jl#739] [Ideas]
After the [JuMP-dev](https://jump.dev/meetings/jumpdev2024/) workshop there are some ideas to explore in TulipaEnergyModel:

- #257 
- #689
- Test [ParametricOptInterface.jl](ParametricOptInterface.jl) for the rolling horizon feature in Tulipa. And maybe for some scenario update in the model (i.e., GUSS from GAMS)
- Test [SparseVariables.jl](https://github.com/sintefore/SparseVariables.jl) to create variables more efficiently
- Test [PiecewiseAffineApprox.jl](https://github.com/sintefore/PiecewiseAffineApprox.jl) to handle convex (or concave) functions in TulipaEnergyModel

**Comment by clizbe** [comment]
@datejada Didn't you do a toy implementation of the third bullet? If so can you link it here?

**Reply by datejada** [reply]
Yeap, this is the link from JuMP documentation: 

https://jump.dev/JuMP.jl/stable/tutorials/algorithms/rolling_horizon/

**Reply by clizbe** [reply]
Oh I meant you did an implementation in your free time. Or was that something else?

**Reply by datejada** [reply]
I did it to contribute to the JuMP open-source community; so yes, it was in my free time.

**Reply by clizbe** [reply]
Ooooh I didn't know that's where it ended up! Cool it's there!
